spec/ruby_lsp/rubocop/addon_spec.rb

```diff
@@ -182,18 +173,15 @@ def bar
   # Lifted from here, because we need to override the formatter to RuboCop in the spec helper:
   # https://github.com/Shopify/ruby-lsp/blob/4c1906172add4d5c39c35d3396aa29c768bfb898/lib/ruby_lsp/test_helper.rb#L20
```

Comment:
Should this comment still remain? It seems that the nuance has changed.

Reply:
Yeah, the method doesn't have much in common with the original anymore. I removed it